pkg/sfu/downtrack.go

```diff
@@ -453,9 +453,12 @@ func (d *DownTrack) CloseWithFlush(flush bool) {
 
 	d.closeOnce.Do(func() {
 		Logger.V(1).Info("Closing sender", "peer_id", d.peerID, "kind", d.kind)
+		d.receiver.DeleteDownTrack(d.peerID)
```

Comment:
should we remove delete from unbind?

Reply:
@cnderrauber I did think about it. But, that is existing behaviour and `Delete` is a no-op if that down track does not exist. So left it in there. Do you see any down side to it?

Reply:
Sounds good. Just think of confused for other people that not familiar with this tricky things.

Reply:
Yeah agreed. I will make an issue to remove it from `Unbind` once we have this code in use for a few days and don't see any issues.